src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let basePrice = days * PRICE_PER_DAY;
```

Comment:
variable basePrice is not changed anywhere, so it supposed to be a const
```suggestion
  const basePrice = days * PRICE_PER_DAY;
```

Reply:
Changed